bot.py

```diff
@@ -16,7 +16,11 @@
 setup_logger(config.log_level)
 
 def main():
-    bot = commands.Bot(command_prefix=config.bot_prefix)
+    intents = discord.Intents.default()
+    intents.message_content = True
+    intents.members = True
+
+    bot = commands.Bot(command_prefix=config.bot_prefix, intents=intents)
 
     bot.help_command = EmbedHelpCommand()
```

Comment:
What's being done in place of !help? I see some of it is built into commands now but how does that work with slash groups?